_rules/table-headers-attribute-refer-to-data-cells-a25f45.md

```diff
@@ -0,0 +1,287 @@
+---
+id: a25f45
+name: Headers attribute of td element refers to cells in the same table element
+rule_type: atomic
+description: |
+  This rule checks that the headers attribute of `td` elements refer to cells in the same `table` element that have a semantic role of `columnheader` or `rowheader`.
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+authors:
+  - Jey Nandakumar
+  - Audrey Maniez
+---
+
+## Applicability
+
+This rule applies to any [`headers`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/td#attr-headers) attribute specified on a [`cell`](https://www.w3.org/TR/html52/tabular-data.html#cell) in a [table](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table) element that is [included in the accessibility tree](#included-in-the-accessibility-tree).
+
+## Expectation
+
+The `headers` attribute of each target element is [a set of space separated IDs](https://www.w3.org/TR/html50/infrastructure.html#set-of-space-separated-tokens), each of which is an ID of an element in the same [document tree](https://www.w3.org/TR/dom41/#document-trees) or [shadow tree](https://www.w3.org/TR/dom41/#shadow-trees) as the applicable elment, that:
+
+1. has a [semantic role](#semantic-role) of `columnheader` or `rowheader`, and
+2. is a [`cell`](https://www.w3.org/TR/html52/tabular-data.html#cell) of the same [`table`](https://www.w3.org/TR/html50/tabular-data.html#concept-table) as the target element
+
+## Assumptions
+
+- This test assumes that the `headers` attribute is only used to identify table headers. If other information is included in the `headers` attribute, the rule may fail on issues that are not accessibility concerns. For example, if `headers` is used to include information for script, this rule may not be accurate.
+- This test assumes that if the IDs of the `headers` attribute, do not live up to the expectations in this rule, the browser will try to calculate a default header from the [algorithm for assigning header cells](https://html.spec.whatwg.org/multipage/tables.html#header-and-data-cell-semantics).
```

Comment:
I think it's actually the opposite. We're assuming that if `headers` attribute fails, the browser won't be able to come up with the correct headers. Also, use "expectation" rather than "expectations", since there's only one of them.